deploy/roles/postgresql-server/tasks/main.yml

```diff
@@ -229,14 +241,14 @@
 
 - name: Install requirements for postgresql Ansible tasks
   yum:
-    name: "{{ item }}"
+    name: [
```

Comment:
Same question here as above re: list syntax

Reply:
(See my previous reply about the deprecation warning.)